src/screens/conclusion/index.js

```diff
@@ -44,43 +41,71 @@ export function Conclusion() {
         <br />
         <h3 className="mt-10 font-brand text-4xl text-primary">Thank you.</h3>
       </div>
-      <div className="flex flex-row justify-evenly p-5">
-        <div className="flex flex-row">
-          <PatreonIcon fill="white" />
-          <div className="mx-4 flex-col">
-            <h5>Please consider supporting our work!</h5>
-            <div className="text-sm">
+
+      <div className="relative mt-24 flex flex-row justify-evenly bg-white bg-opacity-10 py-10">
+        <img
+          className="not-sr-only absolute right-1/4 top-[-230px] max-w-lg lg:w-full"
+          src="/images/conclusion/pups.png"
+          alt="Bear pups"
+        />
+        <div className="flex">
+          <HealthcareIcon />
+          <div className="ml-8 flex-col">
+            <h5 className="mb-2 text-xl font-bold text-white">
+              Please consider supporting our work!
+            </h5>
+            <span className="text-md font-light text-white">
               This website would not be possible without your support. Thank
               you!
-            </div>
+            </span>
           </div>
         </div>
-        <span className="uppercase">Support Us</span>
+        <a
+          href={links.veganHacktivistsSupport}
+          target="_blank"
+          rel="noreferrer"
+          className="self-center rounded-xl bg-primary px-5 py-1 text-xl font-medium uppercase"
+        >
+          Support Us
+        </a>
       </div>
-      <div className="flex flex-row">
-        <h5 className="px-20">Credits and sources</h5>
-        <div>
-          <p>
-            Mauris convallis arcu amet, sollicitudin gravida gravida platea.
-          </p>
-          <p>
-            Velit rhoncus pellentesque feugiat aenean tincidunt. Diam tincidunt
-            aliquet mauris lorem.
-          </p>
-          <p>
-            Egestas scelerisque ac risus. Neque nibh tellus elementum vulputate
-            auctor
-          </p>
-          <p>Arcu amet, sollicitudin gravida gravida platea.</p>
-          <p>Rrhoncus pellentesque feugiat aenean tincidunt</p>
-          <p>
-            Diam tincidunt aliquet mauris lorem egestas scelerisque ac risus
-          </p>
-          <p>Neque nibh tellus elementum vulputate auctor.</p>
-          <p>
-            Mauris convallis arcu amet, sollicitudin gravida gravida platea.
-          </p>
+
+      <div className="mx-auto flex max-w-screen-2xl justify-evenly p-14">
+        <div className="mr-20 flex flex-col">
+          <div className="flex flex-row text-white">
+            <h5 className="mr-16 w-[170px] min-w-[170px] font-bold">
```

Comment:
Not sure if there's another way, but using `px` here to ensure both "Special thanks" and "Credits and sources" stay the same width always so the text lines up together.